### PR TITLE
Guide about processing page builder content

### PR DESCRIPTION
**Pull request by mburridge**
As per issue #126

### CODE REVIEW

File: docs/guides/processing-page-builder-content.md [hunk 1]
```diff
@@ -0,0 +1,365 @@
+# How to process page-builder content in Frontity
+
+Video: https://www.youtube.com/watch?v=qOfENWKR7EE&list=PLC9teX20GdrTBeOzSwE-bFW-MbBEUwowS&index=6&t=159s
```
Comment (juanmaguitar) [inline docs/guides/processing-page-builder-content.md:3]: What is the purpose of this link?

Comment (mburridge) [reply docs/guides/processing-page-builder-content.md:3]: Oh, that's just there for my reference - it will be removed before we publish it.

**Comment by juanmaguitar** [comment]
@mburridge Great content!
I've added a commit to use the proper heading levels, adding a Table of Contents and fixing terminology according to our [textlint rules](https://github.com/frontity/docs/wiki/Textlint)

The article is a bit long in my opinion and I think it would fit better as a Blog Post than as a guide. What do you think if we publish it as a Blog Post instead than as a guide?

**Review by juanmaguitar — APPROVED** [review]

File: docs/guides/processing-page-builder-content.md [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+#### Elementor classes
+
+A complication that needs to be taken into account is that Elementor adds classes to the `<body>` tag. However, you can add these same classes to the `<body>` tag of your Frontity project using the [`<Head>` component](https://api.frontity.org/frontity-packages/core-package/frontity#head).
+
+First import the `<Head>` component into your theme's `index.js` file and then use it in your main `<Theme>` component passing it a `bodyAttributes` prop containing an object with the classes that you want in your `<body>` tag.
+
+```js
+import { Head } from "frontity";
+
+onst Theme = ({ state }) => {
```
Comment (juanmaguitar) [inline docs/guides/processing-page-builder-content.md:257]: There's a typo here in `const`

**Review by juanmaguitar — APPROVED** [review]

File: docs/guides/processing-page-builder-content.md (hunk 2 above)
Comment (mburridge) [reply docs/guides/processing-page-builder-content.md]: Fixed.

**Pull request merged**